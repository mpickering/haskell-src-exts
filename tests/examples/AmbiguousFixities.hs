f = (+ 1) . head &&& tail

f = tail &&& (+ 1) . head
