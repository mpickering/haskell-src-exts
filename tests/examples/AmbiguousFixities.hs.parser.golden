ParseFailed
  (SrcLoc "tests/examples/AmbiguousFixities.hs" 1 5)
  "Ambiguous infix expression: (.){r} and (&&&){l} have precedence 9"
